tests/Api/Responses/Expected/admin/catalog_promotion/post_catalog_promotion_response.json

```
@@ -15,7 +15,7 @@
     "scopes": [
         {
             "@type": "CatalogPromotionScope",
-            "@id": @string@,
+            "@id": "\/api\/v2\/catalog-promotion-scopes\/@integer@",
```

And for other ids of scopes and actions
```suggestion
            "@id": "\/api\/v2\/admin\/catalog-promotion-scopes\/@integer@",
```